File: RotoVR Sample/Assets/Example/Scripts/BLE/Model/DeviceDataModel.cs
using System;

namespace Example.BLE.Model
{
    [Serializable]
    public class DeviceDataModel
    {
        public DeviceDataModel(string name, string address)
        {
            Name = name;
            Address = address;
        }

        public string Name { get; }
        public string Address { get; }
    }
}

File: com.rotovr.android.bleplugin/unitybleplugin/src/main/java/com/rotovr/unitybleplugin/model/DeviceDataModel.java
package com.rotovr.unitybleplugin.model;

import org.json.JSONException;
import org.json.JSONObject;
import java.io.Serializable;


public class DeviceDataModel implements Serializable {

    public DeviceDataModel(String name, String address) {
        Name = name;
        Address = address;
    }

    public String Name;
    public String Address;

    public String toJson() {
        JSONObject obj = new JSONObject();
        try {
            obj.put("Name", Name);
            obj.put("Address", Address);

            return obj.toString();
        } catch (JSONException e) {
            e.printStackTrace();
        }

        return obj.toString();
    }
}
